pkg/api/common/const.go

```diff
@@ -59,9 +59,9 @@ const (
 
 const (
 	// KubernetesDefaultRelease is the default Kubernetes release
-	KubernetesDefaultRelease string = "1.11"
```

Comment:
there might be unit tests that require updating with this change

Reply:
Yes, I pushed the update commit. Also see #878.